src/backend/web/app.js

```diff
@@ -76,6 +76,7 @@ app.use('/', router);
  * Error Handler, Pass to front-end
  */
 /* eslint-disable no-unused-vars */
+// eslint-disable-next-line promise/prefer-await-to-callbacks
```

Comment:
I think it'd be better to actually fix the problem instead of just hiding the warning. We should definitely use async/await instead of callbacks.

Reply:
I thought we can't use async and await in `app.use` so I suppressed the warning. I'll research a bit more to figure out how to solve this. Thanks for your comment